tools/generator-go-sdk/generator/templater_predicates.go

```diff
@@ -94,7 +94,7 @@ func (p predicateTemplater) templateForModel(predicateStructName string, name st
 		for _, fieldName := range fieldNames {
 			fieldVal := model.Fields[fieldName]
 
-			typeInfo, err := golangTypeNameForObjectDefinition(fieldVal.ObjectDefinition)
+			typeInfo, err := golangTypeNameForObjectDefinition(fieldVal.ObjectDefinition, nil) // TODO package path needed?
```

Comment:
at the time of writing these are scoped to the package, but these might become common types (above) - so a TODO either way